packages/inter-protocol/src/vpool-xyk-amm/singlePool.js

```diff
@@ -14,64 +14,62 @@ const getPools = pool => ({
   Secondary: pool.getSecondaryAmount(),
 });
 
-export const singlePool = {
-  allocateGainsAndLosses: (context, seat, prices) => {
-    const { pool } = context.facets;
-    const { poolSeat, zcf, protocolSeat } = context.state;
-    seat.decrementBy(harden({ In: prices.swapperGives }));
-    seat.incrementBy(harden({ Out: prices.swapperGets }));
-    protocolSeat.incrementBy(harden({ Fee: prices.protocolFee }));
+export const makeSinglePool = ammState => {
+  return {
+    allocateGainsAndLosses: (context, seat, prices) => {
```

Comment:
consider,
```suggestion
export const makeSinglePool = ammState => ({
  allocateGainsAndLosses: (context, seat, prices) => {
```
One less indentation

Reply:
done